source/atomics.tex

```diff
@@ -427,6 +427,7 @@
 
 \rSec1[atomics.types.generic]{Atomic types}
 
+\indexlibrary{\idxcode{atomic<\placeholder{type}>}}%
```

Comment:
Maybe index entries for `atomic<`*`integral`*`>` and `atomic<T*>` might make sense, though.